project/plugins.sbt

```diff
@@ -1,13 +1,18 @@
+resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
+resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
+resolvers += Resolver.typesafeRepo("releases")
 
-resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.gov.uk/maven2"
-resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-addSbtPlugin("com.typesafe.sbt"  % "sbt-twirl" 		% "1.5.1")
-addSbtPlugin("com.typesafe.play" % "sbt-plugin"         % "2.8.19")
+addSbtPlugin("uk.gov.hmrc"          % "sbt-auto-build"        % "3.9.0")
+addSbtPlugin("uk.gov.hmrc"          % "sbt-distributables"    % "2.2.0")
+addSbtPlugin("com.typesafe.play"    % "sbt-plugin"            % "2.8.19")
+addSbtPlugin("com.typesafe.sbt"     % "sbt-gzip"              % "1.0.2")
```

Comment:
I'd be careful if you're copying this file over between projects, we've lost sbt-twirl and added gzip here?